MAINTAINERS

```diff
@@ -6,21 +6,32 @@
 # It is structured to be consumable by both humans and programs.
 # To extract its contents programmatically, use any TOML-compliant parser.
 #
-# This file is compiled into the MAINTAINERS file in docker/opensource.
-#
 [Org]
-	[Org."Core maintainers"]
+	[Org."Org maintainers"]
+		people = [
+			"justincormack",
+			"niazfk",
```

Comment:
I'd like to suggest a more diverse group.  Having a wider set of people from a larger community for each project should help the group to build broader consensus.

Trishank, Luke, Dan, Marina, and myself would all be interested and I'm sure other community members would welcome the opportunity too.